ui/app/src/components/Dialogs/ConfirmDialog.tsx

```diff
@@ -24,18 +23,64 @@ import { defineMessages, useIntl } from 'react-intl';
 import StandardAction from '../../models/StandardAction';
 import Dialog from '@material-ui/core/Dialog';
 import { useUnmount } from '../../utils/hooks';
+import { createStyles, makeStyles } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+import confirmGraphicUrl from '../../assets/confirm.svg';
 
 const messages = defineMessages({
-  ok: {
-    id: 'words.ok',
-    defaultMessage: 'Ok'
+  accept: {
+    id: 'words.accept',
+    defaultMessage: 'Accept'
   },
   cancel: {
     id: 'words.cancel',
     defaultMessage: 'Cancel'
   }
 });
 
+const confirmDialogStyles = makeStyles(() =>
+  createStyles({
+    dialog: {
+      '& .MuiPaper-root': {
+        maxWidth: '350px',
+        borderRadius: '20px'
+      }
+    },
+    dialogBody: {
+      backgroundColor: '#fff',
```

Comment:
Don't hard code colours. This disrupts theming. 
Is it actually necessary to set the background? Should have the right background by default.

**When necessary**, make sure to use the `theme` object to set color of text, backgrounds or borders.

Reply:
Dialog Body is using the default background color (gray `#fafafa`)

Reply:
That's not true — check the code.
But, don't use our DialogBody component for the confirm dialog. It has a distinctive style on which dialog body has no value.